addons/l10n_in/tests/test_credit_note_hsn.py

```diff
@@ -0,0 +1,58 @@
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+from odoo import fields, Command
+
+from dateutil.relativedelta import relativedelta
+
+
+@tagged('post_install', '-at_install', 'post_install_l10n')
+class TestCreditNoteHSN(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(chart_template_ref='in')
+        cls.product_line_vals_1 = {
+            'name': cls.product_a.name,
+            'product_id': cls.product_a.id,
+            'account_id': cls.product_a.property_account_expense_id.id,
+            'partner_id': cls.partner_a.id,
+            'product_uom_id': cls.product_a.uom_id.id,
+            'l10n_in_hsn_code': '1234',
+            'quantity': 1.0,
+            'price_unit': 800.0,
+            'price_total': 800.0,
+            'currency_id': cls.company_data['currency'].id,
+            'amount_currency': 800.0,
+            'debit': 800.0,
+            'credit': 0.0,
+        }
```

Comment:
```suggestion
        cls.product_line_vals = {
            'product_id': cls.product_a.id,
            'l10n_in_hsn_code': '1234',
        }
```
Don't need all of those for what we are testing. Also, their is only 1 `product_line_vals` so no need to `_1` it.